perf/benchmark/runner/runner.py

```diff
@@ -170,14 +171,19 @@ def run(self, conn, qps, size, duration):
         if self.mode == "grpc":
             grpc = "-grpc -ping"
 
+        cacert_path = ""
+        if self.cacert is not None:
+            cacert_path = "-cacert {cacert_path}".format(cacert_path=self.cacert)
```

Comment:
I think it would be much clearer to put the argument `-cacert` to line179 the forito run command. Since `cacert_path` should be the path to your cacert, not including the `-cacert` flag.

Reply:
@carolynhu I think the reason doing here is that you need to check if cacert is none or not first, if none then it would not pass in the argument and param altogether

Reply:
Yes, thats the reason but the variable name is actually terrible, should be more `cacert_arg` instead of the current name.